ocaml/database/redo_log.ml

```diff
@@ -37,11 +37,12 @@ let get_static_device reason =
   R.debug "Found %d VDIs matching [%s]" (List.length vdis) reason ;
   match vdis with [] -> None | hd :: _ -> hd.Static_vdis_list.path
 
-(* Make sure we have plenty of room for the database *)
-let minimum_vdi_size =
+let mib megabytes =
```

Comment:
The argument could be made an `int` to simplify the call site and converted to `int64` here.